src/sass/styles-minimal.scss

```diff
@@ -0,0 +1,18 @@
+@charset 'UTF-8';
```

Comment:
Just to mention as per issue #13 that I suggest this file be called `styles-critical.scss` (with the appropriate compiled version name)

Reply:
Agreed